### PR TITLE
fix(): fix pack

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/app/components/app-team/app-team.component.ts [hunk 1]
```diff
@@ -31,6 +35,8 @@ export class AppTeamComponent implements OnInit {
     this.addUser = false;
     this.users = [];
     this.user = auth.getData();
+    this.userForm = {
+      email: null, fullname: null, password: null, confirmPassword: null, admin: 0 };
```
Comment (jkuri) [inline src/app/components/app-team/app-team.component.ts]: } on newline please.

File: src/app/components/app-team/app-team.component.ts [hunk 2]
```diff
@@ -76,6 +77,9 @@ export class AppTeamComponent implements OnInit {
         this.success = false;
       }
     });
+
+    this.userForm = {
+      email: null, fullname: null, password: null, confirmPassword: null, admin: 0 };
```
Comment (jkuri) [inline src/app/components/app-team/app-team.component.ts]: here too.

Comment (Izak88) [reply src/app/components/app-team/app-team.component.ts]: done

File: src/app/components/app-team/app-team.component.ts (hunk 1 above)
Comment (Izak88) [reply src/app/components/app-team/app-team.component.ts]: done

**Pull request merged**